jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -61,6 +61,9 @@ mkdir -p "${LOGS_DIR}/qemu"
 sudo sh -c "cp -r /var/log/libvirt/qemu/* ${LOGS_DIR}/qemu/"
 sudo chown -R ${USER}:${USER} "${LOGS_DIR}/qemu"
 
+mkdir -p "${LOGS_DIR}/cluster-api-config"
+cp -r "/home/airshipci/.cluster-api/*" "${LOGS_DIR}/cluster-api-config/"
```

Comment:
```suggestion
cp -r "/home/airshipci/.cluster-api/." "${LOGS_DIR}/cluster-api-config/"
```
This will also catch hidden files. Maybe not particularly likely in ~/.cluster-api, but seems to me a better general practice to get an accurate dump.